lms/templates/mpesa/mpesa_form.html

```diff
@@ -42,12 +42,11 @@ <h3>For payment, enter the phone number and unlock the phone</h3>
 function validate_int(myEvento) {
   var myCaja = document.getElementById("phone");
   myText = myCaja.value;
-  if ((myEvento.charCode >= 48 && myEvento.charCode <= 57) && myText.length < 12) {
-    dato = true;
-  } else {
-    dato = false;
+  charCode = myEvento.charCode
+  if (charCode < 32 || (charCode >= 48 && charCode <= 57) && myText.length < 12){
```

Comment:
`charCode < 32` - is not clear enough statement could you please clarify it or change to some straightforward solution.

Reply:
I changed the statement to a straightforward solution.  I tested this in Chrom, Firefox and Safari browsers. Only Firefox needs `charCode ==0`. Other browsers don't need this.